python/tank/commands/validate_config.py

```diff
@@ -84,6 +84,8 @@ def _run(self, log, parameters):
     
         # validate environments
         for env_name in parameters["envs"]:
+            log.info("")
+            log.info("Processing environment %s.yml" % env_name)
```

Comment:
OK So cache apps will now output something like

```
Environment asset
------------------------------------------
Environment path:
/sg_toolkit/greyhounddev.shotgunstudio.com/configs/sloth_dev/config/env/asset.yml

Engine tk-nuke - OK!
App tk-multi-workfiles2 (Engine tk-nuke) - OK!
App tk-multi-about (Engine tk-nuke) - OK!
App tk-multi-screeningroom (Engine tk-nuke) - OK!
...
```

It seems a bit of a shame that we get the path only if the yml can be successfully loaded, but I can't see any way around that without some more significant work. Getting the environment name alone though is an improvement on not getting anything.

I have also added this to the updates command code, as this suffered from the same issue.
This has meant a slight duplication of logging in two places, but I feel the compromise is fair due to the being able to see the environment if a yml file could not be read/parsed is a greater gain than the de-duplication of code.